core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/system/SystemUserRoleManager.java

```diff
@@ -45,8 +46,8 @@
     private static Log log = LogFactory.getLog(SystemUserRoleManager.class);
     int tenantId;
     private DataSource dataSource;
+    private static final String SHA_1_PRNG = "SHA1PRNG";
```

Comment:
Shall we make this public and move it to the class where all the constants are defined ?